arches_querysets/querysets.py

```diff
@@ -206,6 +168,55 @@ def _clone(self):
         clone._as_representation = self._as_representation
         return clone
 
+    def _get_node_value_for_python_annotation(self, tile, node, tile_val):
+        datatype_instance = DataTypeFactory().get_instance(node.datatype)
+
+        if self._as_representation:
+            snake_case_datatype = node.datatype.replace("-", "_")
+            if repr_fn := getattr(
+                datatype_transforms,
```

Comment:
This datatype transform module is an arches-app level place to stash better datatype methods without touching core.